lib/location/method_cloud_location.c

```diff
@@ -46,9 +50,7 @@ static void method_cloud_location_positioning_work_fn(struct k_work *work)
 	struct lte_lc_cells_info *scan_cellular_info = NULL;
 	int err = 0;
 #if defined(CONFIG_LOCATION_METHOD_WIFI)
-	struct k_sem wifi_scan_ready;
-
-	k_sem_init(&wifi_scan_ready, 0, 1);
+	k_sem_reset(&wifi_scan_ready);
```

Comment:
An empty line after the variable declarations would be nice.

Reply:
Added. Seems compliance was happy when there was #if statement in the next line after the declarations.